<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CalibrationDialog</class>
 <widget class="QDialog" name="CalibrationDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>890</width>
    <height>540</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>890</width>
    <height>540</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>30000</width>
    <height>30000</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>SL Calibration</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>false</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="3" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Cols:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="colsSpinBox"/>
     </item>
    </layout>
   </item>
   <item row="0" column="0" rowspan="8">
    <widget class="VideoWidget" name="videoWidget" native="true">
     <property name="enabled">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <property name="topMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Pattern:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="patternComboBox">
       <item>
        <property name="text">
         <string>circles</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>checkers</string>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item row="8" column="0">
    <widget class="QPushButton" name="snapButton">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>Snap</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Spacing (mm):</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spacingSpinBox">
       <property name="value">
        <number>0</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="7" column="1">
    <widget class="QPushButton" name="calibrateButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Calibrate</string>
     </property>
    </widget>
   </item>
   <item row="8" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="text">
        <string>&amp;Cancel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="saveButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>&amp;Save</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="1">
    <widget class="QListWidget" name="listWidget">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>16777215</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Rows:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="rowsSpinBox"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>VideoWidget</class>
   <extends>QWidget</extends>
   <header>VideoWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>cancelButton</sender>
   <signal>clicked()</signal>
   <receiver>CalibrationDialog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>703</x>
     <y>498</y>
    </hint>
    <hint type="destinationlabel">
     <x>628</x>
     <y>510</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
